pkg/bazel/BUILD.bazel

```diff
@@ -34,3 +35,19 @@ go_proto_library(
     proto = ":bazel_proto",
     visibility = ["//visibility:public"],
 )
+
+# Need a way to reference the copy of flags.pb.go which lives in bazel-out
+# We can't use a Label because that would resolve to the file at the same path
+# in the source tree.
+filegroup(
+    name = "go_generated_srcs",
+    srcs = [":bazel_go_proto"],
+    output_group = "go_generated_srcs",
+)
+
+write_source_files(
+    name = "write_flags_pb",
+    files = {
+        "flags.pb.go": ":go_generated_srcs",
```

Comment:
Sadly this trick won't work when there are two proto files in this directory.

@kormide any other suggestions?

Reply:
I don't quite follow. Do you mean that `go_generated_srcs` is a filegroup of more than one file, and so `write_source_files` doesn't know which one to write?

Reply:
Or is this that you can't have the same label for the source/output file, so you're putting it in a filegroup? In that case could you just copy the file in the output dir to a different name, then reference that in the dict value?

Reply:
Yeah rules_go will produce more than one file in that output group, and we don't control the names of the files it writes. We want to copy all the output files to the current directory rather than to a new nested subfolder.

Reply:
Hm, what if we supported a wildcard in the macro like:
```
files = {
  "*": ":go_generated_srcs",
}
```
The macro would recognize it as a macro and not pass it into the rule as a label. The rule would somehow know to look at the DefaultInfo of `:go_generated_srcs` and copy the files to the current output dir.

Reply:
Could you play with this one a bit from Paul Johnston:
https://github.com/stackb/rules_proto/blob/7c95feba87ae269d09690fcebb18c77d8b8bcf6a/rules/proto_compiled_sources.bzl
it calls through to his "gencopy" rule, and from a quick read I'm not sure if it supports this case.

Reply:
we have a solution for this now in bazel-lib 0.5.3 using default_output_files rule